File: frameworks/Java/fair/java-kora-jdbc/src/main/java/io/techempower/benchmark/kora/model/Fortune.java
package io.techempower.benchmark.kora.model;

import io.koraframework.database.common.annotation.Column;
import io.koraframework.database.common.annotation.Id;
import io.koraframework.database.jdbc.EntityJdbc;

@EntityJdbc
public record Fortune(@Id @Column("id") int id,
                      @Column("message") String message) {
}

File: frameworks/Kotlin/kora/kora-jdbc-coroutines/src/main/kotlin/io/koraframework/benchmark/model/Fortune.kt
package io.koraframework.benchmark.model

import ru.tinkoff.kora.database.common.annotation.Column
import ru.tinkoff.kora.database.common.annotation.Id
import ru.tinkoff.kora.database.jdbc.EntityJdbc

@EntityJdbc
data class Fortune(
    @field:Id @field:Column("id") val id: Int,
    @field:Column("message") val message: String,
)
